docs/rfcs/1013-parsec-account.md

```diff
@@ -26,485 +36,771 @@ That is how we came to the idea of storing the device on a remote server.
 
-### General principles
+Similarly, we want to be provide an option "remember me" that would store the Parsec Account
+password in the OS keyring.
 
-An authentication service (Parsec Account) would be used to store user devices.
-This service will be requested before connecting to the metadata server (Parsec server) in order to retrieve a specific user device.
-The device will then be used to authenticate to the metadata server.
+> [!NOTE]
+> We might want to improve this in the future by using a dedicated
+> authentication method relying on a proper secret key. This way we would avoid
+> storing the password.
 
-<!-- Device stored on a third-party service -->
+TODO: Discuss Parsec Account offline login.
+
+### 2.5 - Account Devices Vault for server-side stored devices keys
+
+A key feature of Parsec Account is the possibility to avoid device-to-device
+enrollment:
+
+- The user connects to Parsec Account from a new machine.
+- The user can then select an identity (organization ID + User ID) to connect to.
+  Behind the scene, those identities correspond to already existing device (aka
+  Registration Device) that are stored server-side.
+- The existing device is then used to create a new device for the machine.
+
+<!-- Registration Device workflow -->
 ```mermaid
 sequenceDiagram
   autonumber
   participant Client
-  participant S_Acc as Account server
-  participant S_Data as Metadata server
+  participant S_Acc as Server Account APIs
+  participant S_Org as Server Organization APIs
+
+  S_Acc ->> Client: List available identities
+  S_Acc ->> Client: Retrieve the Registration Device
+  Client ->> S_Org: Create new device
+  Client ->> S_Org: Exchange with the server using new device
+```
+
+This feature however requires the need to store the Registration Devices securely
+on the server.
+
+This feature is described in detail in [RFC 1014](1014-account-vault-for-device-stored-on-server.md)
+(for the storing part) and [RFC 1015](1015-registration-device.md) (for how the registration device works).
 
-  Client ->> S_Acc: Retrieve the device of the user
-  S_Acc ->> Client: Device of the user
+## 3 - Datamodel
 
-  Client ->> S_Data: Exchange with the server
+### 3.1 - Glossary
+
+- **Parsec Account**: The generic name of this new service.
+- **Account**: The user's account inside Parsec Account.
+- **Authentication Method**: A way to authenticate to an Account. Each account can have
+  multiple authentication methods (i.e. password, FIDO2 etc.).
+- **Vault**: A pool of encrypted data accessible to a group of authentication methods.
+  Each account can have multiple Vaults (in case of vault key rotation, or if the account
+  access has been reset due to user losing all his authentication methods).
+- **Vault item**: A piece of data that is encrypted by the Vault key. This is typically
+  an Registration Device (see [RFC 1015](1015-registration-device.md)).
+
```

Comment:
```suggestion
- **Identity**: couple organization ID + user ID. An account can store access to multiple identities.

```

I'm not sure about the "can store access" formulation. Maybe "contains" works better.